CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
Hey @prabhakar267 
Sorry, completely missed out on this. Will merge this for now. Let's decide on how to proceed with the fix?